app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  include Pagy::Frontend
+  
+  def sortable(column, title = nil, temp)
+    title ||= column.titleize
+    css_class = column == sort_column(temp) ? "current #{sort_direction}" : nil
```

Comment:
Update it to css_class =  "current #{sort_direction}" if column == sort_column(temp)